lib/BinLib.sol

```diff
@@ -213,6 +195,10 @@ library BinLib {
     }
   }
 
+  function toNearestTick(Bin bin, uint tickSpacing) internal pure returns (Tick) {
```

Comment:
There's no approximation happening, so `Nearest` seems misleading.
```suggestion
  function toTick(Bin bin, uint tickSpacing) internal pure returns (Tick) {
```

Reply:
Introduced bin-tick conversion asymmetry + uniform with olKey methods in 8ef390398905f50d1d11d6462ed559fe86b0a90b